src/assets/scss/gds/template/partials/_fonts.scss

```diff
@@ -33,10 +33,10 @@
 }
 
 @font-face {
-  font-family: 'dotsfont';
-  src: url('#{$fonts-dir}dotsfont.eot');
-  src: url('#{$fonts-dir}dotsfont.eot?#iefix') format('embedded-opentype'),
-       url('#{$fonts-dir}dotsfont.woff') format('woff'),
-       url('#{$fonts-dir}dotsfont.ttf') format('truetype'),
-       url('#{$fonts-dir}dotsfont.svg#dotsfontregular') format('svg');
+  font-family: 'text-security-disc';
+  src: url('#{$fonts-dir}text-security-disc.eot');
+  src: url('#{$fonts-dir}text-security-disc.eot?#iefix') format('embedded-opentype'),
+  url('#{$fonts-dir}text-security-disc.woff') format('woff'),
+  url('#{$fonts-dir}text-security-disc.ttf') format('truetype'),
+  url('#{$fonts-dir}text-security-disc.svg#text-security') format('svg');
```

Comment:
The SVG file prevents build, and is not really a required format.
Webpack will error when running `npm build production` when parsing this SVG.
Please remove the reference to it and the .svg font.